mlflow/tracking/client.py

```diff
@@ -129,11 +153,38 @@ def create_run(self, experiment_id, start_time=None, tags=None):
         Unlike :py:func:`mlflow.start_run`, does not change the "active run" used by
         :py:func:`mlflow.log_param`.
 
-        :param experiment_id: The ID of then experiment to create a run in.
+        :param experiment_id: The string ID of the experiment to create a run in.
         :param start_time: If not provided, use the current timestamp.
         :param tags: A dictionary of key-value pairs that are converted into
                      :py:class:`mlflow.entities.RunTag` objects.
         :return: :py:class:`mlflow.entities.Run` that was created.
+
+        .. code-block:: python
+            :caption: Example
+
+            from mlflow.tracking import MlflowClient
+
+            # Create a run with a tag under an experiment whose default id is '0'.
```

Comment:
How about:
```
# Create a run with a tag under the default experiment (whose id is 0).
```

?

Reply:
That should work too, as it's more precise: there is only "the default" experiment, using the id 0 as the parenthetical clause is additional info.